### ISSUE TITLE
Ensure archived blog images are hosted locally and properly linked to

### ISSUE THREAD

**Issue by TimidRobot** [creativecommons/creativecommons.github.io-source#134]
# Describe the bug
There are a lot of archived blog entries with remote images. Any with a URL that starts with `http://labs.creativecommons.org/` should be hosted from the same directory as the blog entry.

# Related  Links
- [Archive: CC Technical Blog (2007-2014) — Creative Commons on GitHub](https://opensource.creativecommons.org/archives/old-tech-blog/entries/)

**Comment by ritesh-pandey** [comment]
Can I take this up? You want all remote images (`http://labs.creativecommons.org/`) to be part of repository, right?

**Comment by kgodey** [comment]
@ritesh-pandey yes, exactly. go ahead!

**Comment by TimidRobot** [comment]
Note that many of the images are already located in the same directory as the post, but are not being linked to from the post.

**Comment by ritesh-pandey** [comment]
I searched `archived` blogs for links beginning with `labs.creativecommons`. I found two posts where images used these remote URLs.
- content/archives/old-tech-blog/entries/flickr-image-re-use-for-openofficeorg-demo-availlable/contents.lr
- content/archives/old-tech-blog/entries/flickr-image-re-use-for-openofficeorg-new-updates/contents.lr

I made changes in these posts and raised PR.

There is another interesting thing I found. In some posts, image is wrapped inside an anchor tag. Although the image is referenced from local folder, the anchor references remote URL. Should we plan to modify that as well?
